scripts/deps/config/besu/key

```diff
@@ -0,0 +1 @@
+0xc09382acbecd8df7a00899e87bec477958aa11df11d88e552aeec2a3069b9cca
```

Comment:
Same as above. Use Orchestrate configs for better compaitibility

Reply:
I don't think there's any compatibility issue here but consistency may be nice.

Reply:
https://gitlab.com/ConsenSys/client/fr/core-stack/orchestrate/-/tree/master/scripts/besu/config

Reply:
It should not create any compatibility issue. And if we experience any, I commit to fix it.
Also, a private key is usually tied to only one node. Two nodes can't have the same one in a network and it's usually kept private and not shared. So in real life there's no straightforward use case that I can think of for having the same private key even on two identically configured nodes as soon as they are two distinct instances. Ideally I would even let Besu generate a new key each time but it would require some scripting to update the validator extra data in genesis and it's not really useful here IMO.